_posts/2018-12-14-rediscovering-hosts-via-grub2.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Rediscovering hosts locally via Grub2
+date: 2018-12-14 12:25:27
```

Comment:
Please update the publication date to today ;)
```suggestion
date: 2019-01-15 12:25:27
```

Reply:
Done.